config/initializers/new_framework_defaults_6_0.rb

```diff
@@ -0,0 +1,45 @@
+# Be sure to restart your server when you modify this file.
```

Comment:
Worth adding in the commit message that you intend to edit this file in subsequent commits.

Reply:
I've put a note about this in the most recent commit. Do you want one in the commit that introduces this code as well?

Reply:
That's where I meant, yeah - sorry!